platform/event/gossiped_event.proto

```diff
@@ -0,0 +1,61 @@
+/**
+ * # Gossiped Event
+ * An event that is sent and received via gossip
+ *
+ * ### Keywords
+ * The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+ * "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
+ * document are to be interpreted as described in [RFC2119](https://www.ietf.org/rfc/rfc2119).
+ */
+syntax = "proto3";
+
+package com.hedera.hapi.platform.event;
+
+/*
+ * Hedera Network Services Protobuf
+ *
+ * Copyright (C) 2018 - 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import "event_payload.proto";
+import "core_event_data.proto";
+
+option java_package = "com.hedera.hapi.platform.event";
+// <<<pbj.java_package = "com.hedera.hapi.platform.event">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+/**
+ * An event that is sent and received via gossip
+ */
+message GossipedEvent {
```

Comment:
I believe `GossipedEvent` was chosen to avoid a collision with the existing platform `GossipEvent` class. I agree, though, that `Gossiped` is a bit awkward

@lpetrovic05 Could you weigh in here?

Reply:
Thinking some more on this, I think we ought to call the protobuf `GossipEvent`, and rename the Platform class currently called by that name as soon as convenient.

At the end of the day, THIS *is* a gossip event.. Whatever we wrap around the protobuf for internal usage is by definition *not* a gossip event.

Reply:
I also prefer `GossipEvent`. @lpetrovic05 can you remember why the name `GossipedEvent` was chosen over `GossipEvent`? I remember the conversation but not the conclusion.

Reply:
GossipEvent makes sense to me

Reply:
Thusly renamed